doc-site/docs/reference/types/_includes/ffbigint_description.md

```diff
@@ -7,5 +7,49 @@ strings (with base 10).
 On input you can provide JSON string (string with an `0x` prefix are
 parsed at base 16), or a JSON number.
 
-Be careful when using JSON numbers, that the largest
-number that is safe to transfer using a JSON number is 2^53 - 1.
\ No newline at end of file
+## Maximum size of numbers in versions of FireFly up to `v1.3.1`
```

Comment:
Looks good but I wouldn't only had this section in the ffbigint description as it's quite nested down in the documentation under reference. We need to document this in two other places:
- Doc around invoking a smart contract through the API
- Release notes for v1.3.2

Reply:
Thinking something in the interface format https://hyperledger.github.io/firefly/latest/reference/firefly_interface_format/#schema

Reply:
For example you can see in the simple invoke EVM contract tutorial we use a number https://hyperledger.github.io/firefly/latest/tutorials/custom_contracts/ethereum/#invoke-the-smart-contract